GRMustache.swift.podspec

```diff
@@ -1,6 +1,9 @@
+version = ENV["RELEASE_VERSION"]
```

Comment:
Wouldn't this mean that the RELEASE_VERSION has to be available at the time of install as well?

Reply:
You’re absolutely right. I will go ahead and revert this change.